### PR TITLE
feat: cli for sending Wallet{Spend}Action transactions

### LINKED ISSUE #5141: feat: create cli for Msg.Wallet(Spend)Action from x/swingset
## What is the Problem Being Solved?

Need a way to create the `Msg.WalletAction` and `Msg.WalletSpendAction` transactions from the command line.

@mfig please elaborate if necessary

### PR DESCRIPTION
**Pull request by JimLarson**
closes: #5141

## Description

Commands for sending `WalletAction` and `WalletSpendAction` transactions.

### Security Considerations

Ensures address and JSON are valid but performs no other checks.

### Documentation Considerations

Needs long description?

### Testing Considerations

Needs some end-to-end testing, currently difficult to automate.

### CODE REVIEW

File: golang/cosmos/x/swingset/client/cli/tx.go [hunk 1]
```diff
@@ -18,6 +18,10 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+const (
+	FlagSpend = "spend"
```
Comment (michaelfig) [inline golang/cosmos/x/swingset/client/cli/tx.go]:
```suggestion
	FlagAllowSpend = "allow-spend"
```

File: golang/cosmos/x/swingset/client/cli/tx.go [hunk 2]
```diff
@@ -117,6 +121,44 @@ func GetCmdProvisionOne() *cobra.Command {
...
+			}
+			err = msg.ValidateBasic()
+			if err != nil {
+				return err
+			}
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	cmd.Flags().Bool(FlagSpend, false, "Send a WalletSpendAction transaction if true")
```
Comment (michaelfig) [inline golang/cosmos/x/swingset/client/cli/tx.go]:
```suggestion
	cmd.Flags().Bool(FlagAllowSpend, false, "Allow the WalletAction to spend assets if specified")
```
My wording's not so great.  Alternatives in this direction would be appreciated.

**Review by michaelfig — APPROVED** [review]
Thanks for doing this based on a napkin sketch. :smile:

The only change I'd suggest is to make `--spend` more like `--allow-spend` or `--authorize-spend` or something.

File: golang/cosmos/x/swingset/client/cli/tx.go (hunk 1 above)
Comment (JimLarson) [reply golang/cosmos/x/swingset/client/cli/tx.go]: Done.

File: golang/cosmos/x/swingset/client/cli/tx.go (hunk 2 above)
Comment (JimLarson) [reply golang/cosmos/x/swingset/client/cli/tx.go]: Done. Wording seems fine except trimmed the redundant "if true" at the end.

**Pull request merged**